kernel/kernel-api/src/main/java/io/delta/kernel/internal/skipping/DataSkippingUtils.java

```diff
@@ -342,9 +342,8 @@ private static Predicate reverseComparatorFilter(Predicate predicate) {
      * </ol>
      *
      * @param schema schema to prune
```

Comment:
nit (sry) update param docs order to match actual params, now that they've been rearranged